src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
+sys.path.insert(0, '../../sipmpdf')
+sys.path.insert(0, '../')
+
+import matplotlib.pyplot as plt
+import src.sipmanalyze.formats as forms
+import awkward as ak
+import numpy as np
+
+from scipy.signal import find_peaks
+import scipy.integrate as integrate
+from scipy.optimize import curve_fit
+
+
+def import_data(file_location,plot=True):
+  container = forms.waveform.waveform_container.from_root(file_location)
+  tint = container.settings.timeintervals
+  vadc = container.settings.adc_val
+  area = ak.sum(container.waveforms[:, 5:30], axis=-1) * tint * vadc * -1
+  amin, amax = ak.min(area), ak.max(area)
```

Comment:
For histogramming, consider using the `hist` package [1]. As this allows interfacing with other packages HEP libraries, as well as make histogram operation more explicit. Such as histogram slicing that is used in latter operations, as well as bin-center value extraction

[1] https://github.com/scikit-hep/hist